<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEditNewOrderText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddOrder">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevious">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoveCurrent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove current</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetOrders">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewOrder">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAllOrders">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All Orders</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
